src/tools/schema2kotlin.ts

```diff
@@ -153,15 +191,24 @@ export class KotlinGenerator implements ClassGenerator {
   singletonSchemaFields: string[] = [];
   collectionSchemaFields: string[] = [];
 
+  refinement = '{ _ ->\n        true\n    }';
```

Comment:
Prefer formatted strings. To enforce indentation, use `leftPad` or `ktUtils.indent`. e.g.
```
`\
{ _ -> 
    true
}`;
```

Reply:
Also: I think the preferred placeholder in Kotlin is `it`, not `_`.

Reply:
let me double check...

Reply:
Nevermind! `_` is preferred for *unused* arguments, [source](https://kotlinlang.org/docs/reference/lambdas.html#underscore-for-unused-variables-since-11)

Reply:
I think I should just drop these newlines actually.